dist/vespa.spec

```diff
@@ -522,6 +522,7 @@ fi
 %dir %attr(-,%{_vespa_user},%{_vespa_group}) %{_prefix}/var/db/vespa/search
 %dir %attr(-,%{_vespa_user},%{_vespa_group}) %{_prefix}/var/db/vespa/tmp
 %dir %attr(-,%{_vespa_user},%{_vespa_group}) %{_prefix}/var/jdisc_container
+%dir %attr(-,%{_vespa_user},%{_vespa_group}) %{_prefix}/var/secure
```

Comment:
Let's use `secure` instead of `run/secure`, since the latter may be affected by whaterver policies are done to `var` (much like the concern was for `var/run`).